SingularityUI/app/components/common/formItems/Checkbox.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import FormField from './FormField';
+
+class CheckBox extends React.Component {
+
+  render() {
+    return (
+      <FormField
+        id = {this.props.id}
+        noFormControlClass = {this.props.noFormControlClass}
+        prop = {{
```

Comment:
Unfortunately that's how FormField is, and FormField is used in a bunch of other places. So I didn't want to be so fast to modify it.